src/decomp.jl

```diff
@@ -132,9 +132,9 @@ ITensor factorization type for a truncated eigenvalue
 decomposition, returned by `eigen`.
 """
 struct TruncEigen{N}
```

Comment:
We'd want to remove this type parameter as well.

Reply:
And in `TruncSVD` as well presumably.

Reply:
Yeah, that too.